src/Components/App/App.js

```diff
@@ -76,4 +77,10 @@ function App() {
   );
 }
 
+App.propTypes = {
```

Comment:
App doesn't take in any props, so I don't think we need this...

Reply:
good catch no prop types for app

Reply:
sounds good making the change now and pushing up